### PR TITLE
Gml 1852 performant doc ingestion pipeline in support ai

### PR DESCRIPTION
**Pull request by luzhoutg**

### CODE REVIEW

File: eventual-consistency-service/app/supportai/util.py [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+                host=milvus_config["host"],
+                port=milvus_config["port"],
+                support_ai_instance=True,
+                collection_name=name,
+                username=milvus_config.get("username", ""),
+                password=milvus_config.get("password", ""),
+                vector_field=milvus_config.get("vector_field", "document_vector"),
+                text_field=milvus_config.get("text_field", "document_content"),
+                vertex_field=vertex_field,
+                drop_old=True,
```
Comment (RobRossmiller-TG) [inline eventual-consistency-service/app/supportai/util.py:116]: drop_old should be false for checked in code. We don't want to accidentally delete someone's indices

**Review by RobRossmiller-TG — CHANGES_REQUESTED** [review]
Looks good except for the one change regarding `drop_old`.  Please delete old code instead of commenting it out.

There's a lot of duplication between this package and graphrag. In the future, it'd be good to have a common set of functions that both loaders can use. For example, `make_headers` is identical in both packages as well as the whole `ecc_util.py` module

**Comment by luzhoutg** [comment]
> Looks good except for the one change regarding `drop_old`. Please delete old code instead of commenting it out.
> 
> There's a lot of duplication between this package and graphrag. In the future, it'd be good to have a common set of functions that both loaders can use. For example, `make_headers` is identical in both packages as well as the whole `ecc_util.py` module

File: eventual-consistency-service/app/supportai/util.py (hunk 1 above)
Comment (luzhoutg) [reply eventual-consistency-service/app/supportai/util.py]: totally. set it to false now.

**Comment by luzhoutg** [comment]
> Looks good except for the one change regarding `drop_old`. Please delete old code instead of commenting it out.
> 
> There's a lot of duplication between this package and graphrag. In the future, it'd be good to have a common set of functions that both loaders can use. For example, `make_headers` is identical in both packages as well as the whole `ecc_util.py` module

removed all comment out code now. I will start to remove the duplicates after merging parker's and my PR to dev. it would be easier due to less conflicts.

**Comment by RobRossmiller-TG** [comment]
> removed all comment out code now. I will start to remove the duplicates after merging parker's and my PR to dev. it would be easier due to less conflicts.

Absolutely agreed! The comment was as much a note for us to come back to as anything.

**Review by parkererickson-tg — APPROVED** [review]

**Review by RobRossmiller-TG — APPROVED** [review]

**Review by billshitg — APPROVED** [review]

**Pull request merged**